### PR TITLE
Link to table of inspector

### LINKED ISSUE #13: Explain check section more, what attrs exist
Explain check section more, what attrs exist

Follow up to https://github.com/akka/akka-http-scala-seed.g8/pull/12#discussion_r132506029

### PR DESCRIPTION
**Pull request by jiminhsieh**
Fix #13 and bump libraries.

### CODE REVIEW

File: docs/src/main/paradox/testing-routes.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ We simply construct a raw `HttpRequest` object and pass it into the route using
 After all, the route is a function of `HttpRequest => Future[HttpResponse]` - here we don't need to explicitly write code
 that's awaiting on the response, it's handled for us.
 
-Inside the check block we can inspect all kinds of attributes of the received response, like `status`, `contentType` and 
+Inside the check block we can inspect [all kinds of attributes](https://doc.akka.io/docs/akka-http/current/routing-dsl/testkit.html?language=scala#table-of-inspectors) of the received response, like `status`, `contentType` and
```
Comment (raboof) [inline docs/src/main/paradox/testing-routes.md]: (we could use [extref](https://developer.lightbend.com/docs/paradox/latest/features/linking.html#extref-directive) here, but not essential

File: src/main/g8/project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=0.13.17
```
Comment (raboof) [inline src/main/g8/project/build.properties]: We should make that sbt1 at some point, but not needed right now, added issue #42

File: docs/src/main/paradox/testing-routes.md (hunk 1 above)
Comment (jiminhsieh) [reply docs/src/main/paradox/testing-routes.md]: Thanks for the indication! I will push another commit later.

File: docs/build.sbt [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 
-scalaVersion := "2.12.2"
+scalaVersion := "2.12.5"
 
 paradoxProperties in Compile ++= Map(
   "snip.g8root.base_dir" -> "../../../../src/main/g8",
   "snip.g8src.base_dir" -> "../../../../src/main/g8/src/main/",
-  "snip.g8srctest.base_dir" -> "../../../../src/main/g8/src/test/"
+  "snip.g8srctest.base_dir" -> "../../../../src/main/g8/src/test/",
+  "extref.akka.scala.base_url" -> "https://doc.akka.io/docs/akka/current/%s",
```
[the remark this answers is not in the document]
Comment (jiminhsieh) [reply docs/build.sbt]: No, we don't need `.scala` at `paradoxProperties`'s key. I just made it explicit. I will update it. 😄

**Comment by raboof** [comment]
Thanks!

**Pull request merged**